src/main/scala/edg_ide/edgir_graph/ElkEdgirGraphUtils.scala

```diff
@@ -1,15 +1,21 @@
 package edg_ide.edgir_graph
 
 import edg.EdgirUtils.SimpleLibraryPath
+import edg.compiler.{Compiler, TextValue}
+import edg.wir.ProtoUtil.ParamProtoToSeqMap
 import edg.wir.{BlockConnectivityAnalysis, DesignPath}
-import edg_ide.EdgirUtils
+import edg_ide.util.EdgirAnalysisUtils
+import edgir.elem.elem
 import org.eclipse.elk.graph.{ElkGraphElement, ElkNode}
 
 import scala.jdk.CollectionConverters._
 
 
 object ElkEdgirGraphUtils {
   import org.eclipse.elk.graph.properties.IProperty
+
+  // Adds the DesignPatnProperty, containing the node's DesignPath, to nodes.
```

Comment:
Is this supposed to spell DesignPathProperty?

Reply:
It definitely was 🙃
Good catch!